### PR TITLE
Make all CI commit pin changes trigger ciflow/inductor

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #103443

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@
 - test/distributed/test_inductor_collectives.py
 - functorch/_src/partitioners.py
 - functorch/_src/aot_autograd.py
-- .ci/docker/ci_commit_pins/triton.txt
+- .ci/docker/ci_commit_pins/**
```
Comment (desertfire) [inline .github/labeler.yml:23]: .github/ci_commit_pins/**  as well?

**Review by desertfire — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Comment by malfet** [comment]
Does it mean that labeller do not understand whildcards?

**Comment by ezyang** [comment]
Good catch! I actually generalized to the wrong one!!

**Pull request closed without merging**